examples/apps/presence-tracker/README.md

```diff
@@ -2,9 +2,17 @@
 
 **_This demo is a work-in-progress_**
 
-**Presence Tracker** is an example that demonstrates how transient state of audience members can be tracked among other audience members using signals. It does so using fluid-framework's `FluidContainer`, `IServiceAudience`, and `Signaler`.
+**Presence Tracker** is an example that demonstrates how the @fluidframework/presence package can be used to share data
```

Comment:
Nit: I always recommend line-breaking along sentence boundaries for ease of reviewing future content changes.

https://github.com/microsoft/FluidFramework/wiki/Markdown-Best-Practices#line-breaks-along-sentence-boundaries